builtin.c

```diff
@@ -133,6 +133,21 @@ XS(XS_builtin_func1_scalar)
     XSRETURN(1);
 }
 
+XS(XS_builtin_trimmed);
```

Comment:
You can probably just add `Perl_pp_trimmed(aTHX);` to the switch in `XS_builtin_func1_scalar` above, it'll take care of emitting the warning etc.